share/templates/08-Orchestral/02-Symphony_Orchestra.mscx

```diff
@@ -1198,16 +1199,19 @@
           <gateTime>100</gateTime>
           </Articulation>
         <Channel name="arco">
+          <controller ctrl="0" value="0"/>
```

Comment:
I'm not sure about those added `controller` tags, not sure whether thay harm or not

Reply:
@jthistle might help here

Reply:
See https://github.com/musescore/MuseScore/pull/5350#pullrequestreview-294890474 for further discussion.